### PR TITLE
Add feature page for Configuration Resource Validation [Beta]

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

**Comment by johnma14** [comment]
@jasonwzm PTAL

**Comment by jasonwzm** [comment]
Thanks Mariam, why was this considered Beta, was it just the default? Why not stable?

**Comment by johnma14** [comment]
@jasonwzm Thanks for taking a look at this. This is marked as beta here: https://istio.io/latest/about/feature-stages/#core. If  this is a stable feature, can you go ahead and take a look at the checkboxes for the stable feature and mark the ones that are applicable and if there is any that is not, just leave a short note. Thanks.

**Comment by brian-avery** [comment]
@jasonwzm gentle reminder. Can you please review?

File: features/config-resource-validation.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [ ] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Stable
+
+### Requirements: 
+
+**Performance**
```
Comment (jasonwzm) [inline features/config-resource-validation.md:199]: Performance tests are done by Kuberenetes. Istio is not conducting performance tests for this.

**Review by jasonwzm — CHANGES_REQUESTED** [review]
We are leveraging much of Kubernetes capability for resource validation. The feature is GA in Kubernetes, therefore we can mark it Stable.

**Review by johnma14 — COMMENTED** [review]
Addressed recommended changes.

File: features/config-resource-validation.md (hunk 1 above)
Comment (johnma14) [reply features/config-resource-validation.md:199]: Done. Thanks @jasonwzm. Appreciate it.

**Pull request closed without merging**